README.arm.md

```diff
@@ -68,16 +68,23 @@ ARMv6 as described in the Build Dependencies section above.
 
 ## Raspberry Pi 2
 
-For Raspberry Pi 2, which is ARMv7, the default build should work. In
-the case of Raspberry Pi 2, in case the default build fails, download
-LLVM binaries from the LLVM website.
+The Raspberry Pi 2 ARM CPU type is also not detected by LLVM.
+Similar to the above instructions for the Raspberry Pi, define the
+`JULIA_CPU_TARGET` variable but set it to `cortex-a7` instead.
+
+Depending on the exact compiler and distribution, there might be a build failure
+due to unsupported inline assembly. In that case, add `MARCH=armv7-a` to
+`Make.user`.
+
+If the build still fails, try and download LLVM binaries from the LLVM website:
```

Comment:
It seemed appropriate because "just download LLVM binaries" probably won't fix every build problem ("if the build still fails") :-) I'll get rid of it!